.buildkite/scripts/setenv.sh

```diff
@@ -2,37 +2,20 @@
 
 set -euo pipefail
 
-REPO="beats"
-TMP_FOLDER="tmp.${REPO}"
-DOCKER_REGISTRY="docker.elastic.co"
-SETUP_GVM_VERSION="v0.5.1"
-DOCKER_COMPOSE_VERSION="1.21.0"
-DOCKER_COMPOSE_VERSION_AARCH64="v2.21.0"
-SETUP_WIN_PYTHON_VERSION="3.11.0"
-NMAP_WIN_VERSION="7.12"           # Earlier versions of NMap provide WinPcap (the winpcap packages don't install nicely because they pop-up a UI)
+WORKSPACE=${WORKSPACE:-"$(pwd)"}
 GO_VERSION=$(cat .go-version)
-ASDF_MAGE_VERSION="1.15.0"
-ASDF_TERRAFORM_VERSION="1.0.2"
-PACKAGING_PLATFORMS="+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64"
-PACKAGING_ARM_PLATFORMS="linux/arm64"
-AWS_REGION="eu-central-1"
-NODEJS_VERSION="18.17.1"
 
-export SETUP_GVM_VERSION
-export DOCKER_COMPOSE_VERSION
-export DOCKER_COMPOSE_VERSION_AARCH64
-export SETUP_WIN_PYTHON_VERSION
-export NMAP_WIN_VERSION
+export REPO="beats"
+export DOCKER_REGISTRY="docker.elastic.co"
+export SETUP_GVM_VERSION="v0.5.1"
+export DOCKER_COMPOSE_VERSION="1.21.0"
+export DOCKER_COMPOSE_VERSION_AARCH64="v2.21.0"
+
+export ASDF_NODEJS_VERSION="18.17.1"
+export AWS_REGION="eu-central-1"
```

Comment:
hopefully, we can get rid of all of the above (and the entire script) in a future PR. The direction is that there should be almost no repo specific "magic" scripts that ran e.g. as pre-command hooks, but instead as much as possible should be in the command, so that it's easy to troubleshoot issues over ssh or rdp with real vms.

Reply:
Will be done in separate PR shortly.